### PR TITLE
Clean prototype

### PR DESCRIPTION
**Pull request by zoebarth**
Add alt tag to get rid of warning, implement functionality upon pressing enter on search bar, improve route view css
image

### CODE REVIEW

File: frontend/src/route/Route.js [hunk 1]
```diff
@@ -12,8 +12,6 @@ function Route({ places: initialPlaces }) {
 
   // referece: https://egghead.io/lessons/react-persist-list-reordering-with-react-beautiful-dnd-using-the-ondragend-callback
   function handleOnDragEnd({ destination, source, draggableId }) {
-    // const  = result;
```
Comment (Cindyzhang977) [inline frontend/src/route/Route.js:15]: thanks :)

File: frontend/src/search/Search.js [hunk 2]
```diff
@@ -3,21 +3,36 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
-import React from 'react';
+import React, { useState } from 'react';
 import styles from './Search.module.css';
+import { useHistory } from 'react-router-dom';
 
 /**
  * Creates Search component with search bar.
  */
 function Search() {
+  const [name, setName] = useState('');
```
Comment (mecolmg) [inline frontend/src/search/Search.js:14]: nit: `query` would probably be better.

File: frontend/src/search/Search.js [hunk 3]
```diff
@@ -3,21 +3,36 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
+      <Form
+        inline
+        className={styles.form}
+        onSubmit={() => {
+          history.push(`/explore?search=${name}`);
+        }}
+      >
+        <FormControl
+          type="text"
+          className="mr-sm-2 "
```
Comment (mecolmg) [inline frontend/src/search/Search.js:31]: nit: remove extra space

File: frontend/src/search/Search.js [hunk 4]
```diff
@@ -3,21 +3,36 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
         <h1>Where to?</h1>
       </div>
-      <Form inline className={styles.form}>
-        <FormControl type="text" className="mr-sm-2" />
-        <Button variant="secondary">
+      <Form
+        inline
+        className={styles.form}
+        onSubmit={() => {
+          history.push(`/explore?search=${name}`);
```
Comment (mecolmg) [inline frontend/src/search/Search.js:26]: Should you validate that the query is not empty first?

**Pull request merged**